pycm/pycm_obj.py

```diff
@@ -880,3 +880,65 @@ def combine(self, other):
         return ConfusionMatrix(
             matrix=matrix_combine(
                 self.matrix, other.matrix))
+
+    def plot(
+            self,
+            normalized=False,
+            one_vs_all=False,
+            class_name=None,
+            title='Confusion Matrix',
+            number_label=False,
+            cmap=None,
+            number_label_cmap=None,
```

Comment:
What's this parameter for ? text color ?
I think it would be better if we choose color from `cmap` automatically
@alirezazolanvari ?

Reply:
It's just for the case that user decides to choose his/her number label color directly.
But if you think it could be confusing I'll remove it.

Reply:
I think it's not a necessary parameter and it's a little vague.
But if you think it will be useful for users keep it ;-)

Reply:
Yes I think you're right :+1: